source/request-response/subscription_manager.c

```diff
@@ -297,26 +307,55 @@ static void s_cull_unused_subscriptions(struct aws_rr_subscription_manager *mana
     aws_hash_table_foreach(&manager->subscriptions, s_rr_subscription_cull_unused_subscriptions_wrapper, manager);
 }
 
-static const char *s_request_response_subscription_event_type_to_c_str(enum aws_rr_subscription_event_type type) {
+static const char *s_rr_subscription_event_type_to_c_str(enum aws_rr_subscription_event_type type) {
```

Comment:
Trivial: In other recent cases where we're returning a readable string, it feels like we don't translate the enum to a prefix-less PascalCase but instead return a normal reading format. e.g. "Request subscribe successful" vs. "RequestSubscribeSuccess". I think that's preferable but it's extremely trivial and doesn't need to be changed but might be better for logs.

Reply:
Have come to prefer camel case for easier searchability, tbh.